### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by aantn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,106 +1,79 @@
...
-  [![slack robusta](https://img.shields.io/badge/Slack-Join-4A154B?style=flat-square&logo=slack&logoColor=white)](https://bit.ly/robusta-slack)
- <a href="https://www.linkedin.com/company/robusta-dev/"><img alt="LinkedIn" title="LinkedIn" src="https://img.shields.io/badge/-LinkedIn-blue?style=flat-square&logo=Linkedin&logoColor=white"/></a>
-  <a href="https://www.youtube.com/channel/UCeLrAOI3anJAfO3BrYVB62Q"><img alt="Youtube" title="Youtube" src="https://img.shields.io/youtube/channel/subscribers/UCeLrAOI3anJAfO3BrYVB62Q?color=%23ff0000&label=Robusta%20Dev&logo=youtube&logoColor=%23ff0000&style=flat-square"/></a>
-
+  <h1 align="center">Robusta - Better Prometheus Alerts (and more) for Kubernetes</h1>
+  <h2 align="center">Enrich, Group, and Remediate your Alerts</h2>
+  <p align="center">
+    <a href="#%EF%B8%8F-how-it-works"><strong>How it Works</strong></a> |
+    <a href="#-installing-robusta"><strong>Installation</strong></a> | 
+    <a href="#installing-robusta"><strong>Integrations</strong></a> |
```
Comment (pavangudiwada) [inline README.md:9]: Link is broken

Comment (pavangudiwada) [inline README.md:9]: Link is broken

File: README.md [hunk 2]
```diff
@@ -1,106 +1,79 @@
...
+Robusta integrates with Prometheus (e.g. `kube-prometheus-stack` or Coralogix) by webhook and adds features like: 
 
-By listening to all the events in your cluster, Robusta can tell you *why* alerts fired, what happened at the same time, and what you can do about it.
+* **Grouping** - reduce notification spam with "Live Summary Messages" and Slack threads 🧵
+* **Enrichment** - see pods logs and other relevant information alongside your alerts
+* **Self-Healing** - define auto-remediation rules for alert response, like running Kubernetes jobs or executing bash commands
+* **Routing by Team/Namespace** to different channels based on labels or Kubernetes metadata
+* **Problem-Detection without PromQL** - generate Kubernetes-native alerts for OOMKills, failing Jobs, and more
+* **Change Tracking** for Kubernetes Resources so you can correlate between alerts and new deployments
+* **Auto-Resolve** for integrations like Slack/Jira, Robusta updates the external system when alerts are resolved
```
Comment (pavangudiwada) [inline README.md:25]: I think it should be. Without the "-" it's hard to make sense of the sentence. 

Auto-Resolve - for integrations like Slack/Jira, Robusta updates the external system when alerts are resolved

**Review by pavangudiwada — APPROVED** [review]
Left minor comments. Very nice.

File: README.md (hunk 1 above)
Comment (aantn) [reply README.md]: Fixed, thanks

File: README.md (hunk 2 above)
Comment (aantn) [reply README.md]: Fixed. You're right.

**Comment by aantn** [comment]
Thanks!

**Pull request merged**